src/Workspaces/Core/Portable/Shared/Extensions/DocumentExtensions.cs

```diff
@@ -60,7 +62,7 @@ public static async Task<SemanticModel> GetSemanticModelForSpanAsync(this Docume
                 return await document.GetSemanticModelAsync(cancellationToken).ConfigureAwait(false);
             }
 
-            var node = token.Parent.AncestorsAndSelf().FirstOrDefault(a => a.FullSpan.Contains(span));
+            var node = token.Parent.AncestorsAndSelf().First(a => a.FullSpan.Contains(span));
```

Comment:
What is this change about?

Reply:
Commit message for f84867d6657c15905558a9472600ec8a3c449f03 describes the issue. Let me know if it's clear.

Reply:
lol, I thought it was more than what it is :)